wagtailtinymce/rich_text.py

```diff
@@ -56,7 +141,7 @@ def getDefaultArgs(cls):
             'options': {
                 'browser_spellcheck': True,
                 'noneditable_leave_contenteditable': True,
-                'language': translation.to_locale(translation.get_language()),
+                'language': translation.to_locale(translation.get_language() or 'en'),
```

Comment:
Just looking at a problem relating to language codes and I think it might be best not to set a default language code here at all but instead leave TinyMCE to provide the default value. With the current code, if you have Django's LANGUAGE_CODE setting is set to a value that does not have a corresponding TinyMCE translation file (https://www.tinymce.com/docs/configure/localization/#language), then the editor will fail to display. For example if the LANGUAGE_CODE is 'en-us' a 404 will occur when TinyMCE tries to load wagtailtinymce/js/vendor/tinymce/langs/en_US.js.

By deferring to TinyMCE's default language at least  everyone gets the editor displayed out of the box.